test/e2e/branding_test.go

```diff
@@ -0,0 +1,71 @@
+package e2e
+
+import (
+	"k8s.io/apimachinery/pkg/util/wait"
+	"testing"
+	"time"
+
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	consoleapi "github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/testframework"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Test the customization features of the config map
+
+// the simpler test first would be branding set via the operator config
+func TestOperatorConfigBranding(t *testing.T) {
+	expectedBrand := operatorsv1.BrandDedicated
+	client := testframework.MustNewClientset(t, nil)
+	originalBrand := testframework.GetBrand(t, client)
+	// defer the cleanup
+	defer testframework.PutBackBrand(t, client, originalBrand)
```

Comment:
Should this just be `defer testframework.SetBrand(orig)` again?  Not sure if `PutBackBrand()` looks like it really adds value?

Reply:
wow, yeah, the function started out more involved but got much simpler as I progressed

Reply:
see https://github.com/openshift/console-operator/pull/206/files#diff-142611e0533ca0e667c4d244cc42c3e4R22

Same pattern is probably sufficient here.